staging/src/k8s.io/legacy-cloud-providers/aws/tags.go

```diff
@@ -94,7 +94,7 @@ func (t *awsTagging) initFromTags(tags []*ec2.Tag) error {
 	}
 
 	if legacyClusterID == "" && newClusterID == "" {
-		klog.Errorf("Tag %q nor %q not found; Kubernetes may behave unexpectedly.", TagNameKubernetesClusterLegacy, TagNameKubernetesClusterPrefix+"...")
+		klog.ErrorS(err,"Tag legacy nor prefix not found; Kubernetes may behave unexpectedly.", "TagNameKubernetesClusterLegacy", TagNameKubernetesClusterLegacy, " TagNameKubernetesClusterPrefix", TagNameKubernetesClusterPrefix+"...")
```

Comment:
I'd change the error message to `Cluster legacy and prefix tags  not found; Kubernetes may behave unexpectedly"